src/main/python/rlbot/parsing/bot_config_bundle.py

```diff
@@ -118,3 +149,19 @@ def get_bot_config_bundles(num_participants, config: ConfigObject, config_locati
             logger.debug("Reading raw config")
 
     return config_bundles
+
+
+def get_script_config_bundles(match_config: ConfigObject, config_location):
+    """
+    Adds all the script config bundles found in the match config file.
+    Attempts to read indexed config keys until it hits an empty one, up to a maximum of 100
+    """
+    config_bundles = []
+    for i in range(100):
```

Comment:
This 100 seems a bit limiting. Would a while loop be better?
```suggestion
    while script_config_relative_path is not None:
```

Reply:
Yeah fair enough, I'll make it a while loop. Using more than 100 scripts is probably a poor idea, but if anyone ever tries, we can probably be entertained!

Reply:
I think 100+ scripts is overkill too but it would be unpleasantly surprising if it just started dropping scripts without an error. So rather than throwing an error, it's probably nicer to do it with a `while`.